src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -1069,17 +1069,39 @@ private void ImportBinaryOperation(ILOpcode opcode)
             {
                 case ILOpcode.add_ovf:
                 case ILOpcode.add_ovf_un:
-                case ILOpcode.mul_ovf:
-                case ILOpcode.mul_ovf_un:
                 case ILOpcode.sub_ovf:
                 case ILOpcode.sub_ovf_un:
                     _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Overflow), "_ovf");
                     break;
+                case ILOpcode.mul_ovf:
+                case ILOpcode.mul_ovf_un:
+                    // Required for ARM
+                    if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.LMulOfv), "_lmulovf");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ULMulOvf), "_ulmulovf");
+                    }
 
+                    _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Overflow), "_ovf");
+                    break;
                 case ILOpcode.div:
                 case ILOpcode.div_un:
                 case ILOpcode.rem:
                 case ILOpcode.rem_un:
+                    // Required for ARM
+                    if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM)
+                    {
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ULMod), "_ulmod");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.LMod), "_lmod");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.UMod), "_umod");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Mod), "_mod");
+
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ULDiv), "_uldiv");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.LDiv), "_ldiv");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.UDiv), "_udiv");
+                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Div), "_div");
+                    }
+
                     // Required for ARM64
```

Comment:
Is this one required for ARM32 too?

Would be nice to update the comment if it is (or to just put it into an explicit `if (_compilation...)` block like you have done for ARM32 - we should have done it for the ARM64 specific helpers too.